views/schedule-form-add.twig

```diff
@@ -136,7 +136,7 @@
 
                         {% set title %}{% trans "Layout" %}{% endset %}
                         {% set helpText %}{% trans "Please select a Layout for this Event to show" %}{% endset %}
-                        {{ forms.dropdown("campaignId", "single", title, campaign.campaignId, [campaign], "campaignId", "campaign", helpText, "layout-control", "", "", "", attributes, "", "", "", readonlySelect) }}
+                        {{ forms.dropdown("campaignId", "single", title, null, null, "campaignId", "campaign", helpText, "layout-control", "", "", "", attributes, "", "", "", readonlySelect) }}
```

Comment:
Could we double check if that breaks the add schedule form from Layout/Campaign pages please?
I think it relies on that being passed from the backend to populate the field (the layout/campaign field is also disabled in this flow)

Reply:
Yes you're right, reverting.